x11/tktable/Portfile

```diff
@@ -4,19 +4,20 @@ PortSystem          1.0
 PortGroup           active_variants 1.1
 
 name                tktable
-version             2.10
-revision            2
+version             2.11
+revision            0
 categories          x11
+license             Tcl/Tk
 maintainers         nomaintainer
 description         A table/matrix widget extension to Tk/Tcl
 long_description    ${description}
 homepage            http://tktable.sourceforge.net/
 platforms           darwin
-master_sites        sourceforge
-distname            Tktable${version}
-checksums           rmd160  b326a472584a6965328f3f27dc98f69ddbdd45f9 \
-                    sha256  c335117fa1be45fe4d3032e96fd4b4641fff6a4f8467878608dabed11198a4cb \
-                    size    284135
+master_sites        https://chiselapp.com/user/pooryorick/repository/tktable/tarball/4018de567f/
```

Comment:
Just for future reference, a comment here as to why this repo is being used would be useful. Something along the lines of the comment in the PR. Otherwise someone looking at this portfile in the future might be a bit confused as to why this is being used.

Reply:
I have added a comment to the Portfile.